docs/content/components/flexitem.md

```diff
@@ -12,6 +12,22 @@ import Example from '../../src/@primer/gatsby-theme-doctocat/components/example'
 Use `FlexItem` to specify the ability of a flex item to alter its
 dimensions to fill available space
 
+## Deprecation
+
+Use [Box](/components/box) instead.
+
+**Before**:
+
+```erb
+<%= render Primer::FlexItemComponent.new(flex_auto: :auto) %>
+```
+
+**After**:
+
+```erb
+<%= render Primer::BoxComponent.new(flex_auto: :auto) %>
```

Comment:
Is this correct? Or would we need `display: :flex` here as well?

Reply:
we don't! `FlexItem` doesn't actually set `display: :flex`
The only thing it did was dealing with `flex_auto:`